chitech_composite_processor/Utils_Info.py

```diff
@@ -0,0 +1,202 @@
+import numpy as np 
+import matplotlib.pyplot as plt
+
+#====================================================================
+def GenerateSpectrumData(neutron_gs, psi, sigma_heat, gamma_gs=[]):

```

Comment:
checkout this for duplicate functions

https://stackoverflow.com/questions/4383571/importing-files-from-different-folder